.codeclimate.yml

```diff
@@ -0,0 +1,162 @@
+# This is our default .CodeClimate.yml, broken out by language. Uncomment the
```

Comment:
I think we get extra credit if ansible-lint is added to .codeclimate.yml. :) I don't know how hard this would be to add, but there would have to be an ansible-lint ignore for any of our Conjur secrets/policy YAML files.

Reply:
Would love to, but running codeclimate like we do, it seems we are limited to a [set of plugins](https://docs.codeclimate.com/docs/list-of-engines) that doesn't include ansible-lint. We'd have to add it directly in the Jenkinsfile.

Reply:
Ansible-Lint can be included in Molecule tests! I files a follow-up issue, #46